src/EnergyPlus/Psychrometrics.cc

```diff
@@ -780,21 +787,21 @@ namespace Psychrometrics {
 
         // If below -100C,set value of Pressure corresponding to Saturation Temperature of -100C.
         if (Tkel < 173.15) {
-            Pascal = 0.0017;
+            Pascal = 0.001405102123874164;
 
             // If below freezing, calculate saturation pressure over ice.
-        } else if (Tkel < DataGlobalConstants::KelvinConv) { // Tkel >= 173.15
-            Real64 const C1(-5674.5359);                     // Coefficient for TKel < KelvinConvK
-            Real64 const C2(6.3925247);                      // Coefficient for TKel < KelvinConvK
-            Real64 const C3(-0.9677843e-2);                  // Coefficient for TKel < KelvinConvK
-            Real64 const C4(0.62215701e-6);                  // Coefficient for TKel < KelvinConvK
-            Real64 const C5(0.20747825e-8);                  // Coefficient for TKel < KelvinConvK
-            Real64 const C6(-0.9484024e-12);                 // Coefficient for TKel < KelvinConvK
-            Real64 const C7(4.1635019);                      // Coefficient for TKel < KelvinConvK
+        } else if (Tkel < DataGlobalConstants::TriplePointOfWaterTempKelvin) { // Tkel >= 173.15, Tkel < 273.16 (0.01°C)
```

Comment:
The actual fix

Reply:
![image](https://user-images.githubusercontent.com/5479063/122747597-8cd52c80-d28b-11eb-9789-cf81f2b5395f.png)